### ISSUE TITLE
Using i18n of releases/builds

### ISSUE THREAD

**Issue by skreutzer** [w3c/epubcheck#498]
How can the i18n of releases/builds be used? Neither the readme nor the Wiki nor the --help command line argument seem to give a hint how to switch to another language.

**Comment by tofi86** [comment]
This isn't supported officially yet...

But You can set it on the command line via JVM parameters:

```
java -Duser.country=DE -Duser.language=de -jar epubcheck.jar ...
```

**Comment by pettarin** [comment]
Perhaps a `--language` (or `--lang` or whatever) command line parameter would be adequate. If not specified, the tool should use the current locale as it does now (which can be specified via `-Duser.language=xx`).

**Comment by tofi86** [comment]
Good idea! :)

**Comment by tofi86** [comment]
This will probably be implemented with PR #650 in near future.
